sdk/core/core-client/src/base64.browser.ts

```diff
@@ -40,3 +40,18 @@ export function decodeString(value: string): Uint8Array {
   }
   return arr;
 }
+
+/**
+ * Converts a uint8Array to a string.
+ */
+export function uint8ArrayToString(ab: Uint8Array): string {
+  return decodeURIComponent(
+    escape(
```

Comment:
I'm confused about the use of `decodeURIComponent` and `escape` here -- especially since `escape` looks deprecated on MDN?

Also if we're trying for unicode support it looks like there are some interesting notes here: https://developer.mozilla.org/en-US/docs/Web/API/btoa#unicode_strings

Reply:
the issue is that the challenge comes encoded